crates/bindings-macro/src/sats.rs

```diff
@@ -52,8 +52,8 @@ pub(crate) fn sats_type_from_derive(
     crate_fallback: TokenStream,
 ) -> syn::Result<SatsType<'_>> {
     let data = match &input.data {
-        syn::Data::Struct(struc) => {
-            let fields = struc.fields.iter().map(|field| SatsField {
+        syn::Data::Struct(s) => {
+            let fields = s.fields.iter().map(|field| SatsField {
```

Comment:
This is not an improvement; `struc` is more clear and readable than just `s`. Please revert this change.